account_reconciliation_widget/models/reconciliation_widget.py

```diff
@@ -1033,6 +1033,7 @@ def _get_statement_line(self, st_line):
             # FIXME: where to fill?
             # 'note': st_line.note or "",
```

Comment:
While we here, I believe this should be `st_line.narration` as inherited from `account.move`. The `note` field was removed in Odoo 14: https://github.com/OCA/OpenUpgrade/blob/14.0/openupgrade_scripts/scripts/account/14.0.1.1/upgrade_analysis.txt#L47

(better to rename the key as well, both here and in the qweb template).